### DISCUSSION TITLE
Life Itself 101: A common core curriculum for Life Itself-ers (& more generally for meta-crisis & second renaissance)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#907] [Ideas]
> [!note]
> 
> This is a preliminary brainstorm with @shiweiart. **Comments, questions, suggestions welcome!**

## Preliminary brainstorm and motivation (where we started)

What are the "fundamentals: that it would be useful for people to *know*[^1] when they come to a residency, an event or a dialog?

[^1]: this is about classic "knowing" rather than about practices or competences - the latter are v important, just not focus here.

Brainstorm:

- Buddhist psychology
  - Attachment to views: why is that at the core of politics and conflicts.
  - Nature of suffering: (not materialistic and *not* pain). 4NT.
  - Buddhist psychology (of mind): seeds metaphor etc.
- Wilber 101
  - GUTO: grand unified theory of ontology (integral psychology)
  - GUTH: grand unified theory of history aka cultural evolution, spiral dynamics etc
- Modernity and its blindspots: materialism, capitalism, secularism etc
  - Shadows of the enlightenment
  - Equality complex
  - Securalism and shadow of religion

Aside: Also really hard to talk about 2nd Renaissance without covering this stuff.

**Conclusion**:  this is basically a "Life Itself 101" aka an "Introduction to key 'views' at Life Itself"

### ⏭️ Next steps in shaping:

- [x] Motivation for doing this ✅ in SCQA below
- [x] Description
- [ ] Skeleton
- [ ] Appetite

### Situation: What's happening ...

- People come to residencies or life itself community
  - NB: all of this is o/c a micro-cosm of something broader. however focus on LI here (see higher level version below for the macrocosm)
- Heard via of word of mouth or online
- We see overlapping "groupings" of views and background reading
  - Some integralists, some vervaek-ites, some mcgilchristers, some shamanists, Harawites (Donna Haraway) etc - see diagram below
- There's no common "core" curriculum that people have consistently
  - The thing closest to common core we see is around integral - but even there quite variable and many people have not heard at all)
- The commonalities that are there are ...
  - Feel something is deeply wrong
  - Seeking a sense of community and belonging
- We (e.g. LI core) don't know of one work (or even several works) that *efficiently* provide a *foundational* (world)views and analysis

### Complication: What's not working about it

- Have a "tower of babel" of competing frames and analyses => inability to "build the tower" (i.e. build the new paradigm)
- We can't just say to someone: read X, Y, Z and see what you think so that we can efficiently a) filter b) start from a common base
  - Spend most of time trying to get to a common base
  - Spend time on people who aren't actually ultimately that aligned (wastes their time and ours)
  - Conversations are often unproductive ...
  - People aren't aligned
- We aren't clear on where Life Itself are similar / distinct from others
  - People often think Life Itself and community are something we are not ... leads to confusion and false +ves and -ves (+ves who become disillusioned are especially important)
- We aren't sure where people are at - we don't even know if they are aligned or not
- Huge amount of repetition going through the basics ...

### Question

What is a Life Itself 101 course including assessment of its effectiveness such that there is (increasingly) a shared understanding of what Life Itself "believes" (foundational views in X, Y, Z way) by a) "core team" (v important. course is prerequisite) b) residents (e.g. course during a residency) c) amongst broader community

## Hypothesis

Brainstorm of potential questions and topics

- Paradigms: onto-social-techno paradigms. aka cultures, institutions and technologies.
- Spiral dynamics color scheme and GUTHs (grand unified theories of history)
- Distinction of waking up, cleaning up, growing up, showing up. i.e. inner development is multi-dimensional.
- What is the source of suffering (e.g. in buddhism): distinguishing pain and suffering etc etc
- Buddhist psychology: e.g. Heart of Buddha's teaching
- Idea of worldviews and that they are surprisingly deep (e.g. changing how visual perception works)
- Modernity: what are the key beliefs of modernity (rationality, individuality, progress, securalism)
- Blindspots: https://lifeitself.org/initiatives/blind-spot-series
- Primacy of being - https://lifeitself.org/primacy-of-being
- Need for action and especially collection action - e.g. intro to basic game theory, political theory (+ve version of the state), 
  - Role of "religion", shared norms and views etc. 

Notes

- 2h a week of watching a video for 2 months = 8 weeks
- 1h a week of dialog a week
- ~24h course
- Before a residency: watch a 2h introduction

---

### Situation: What is happening to the world ...

- Earth is crumbling under our feet
- Deep sense something is going wrong
- Without understanding why
- An era is coming to an end
- To create, we need to understand what is this era dying of and create a common understanding/sensemaking
- Example: this person is into their EA thing, this person is into their "green" indigenous thing (both are valuable ... but ...)
  - Everyone is talking past each other
  - There's not a real sense of why are we suffering from polarization and loneliness.
- Summarize: depression, loneliness, polarization. This surface ... (we agree on) but there a million different reasons they think this is going on.

### Complication

- Need a correct diagnosis to have a correct healing.
- V hard to engage in a conversation when there isn't a common understanding of the why
  - It creates a dispersion of energy.
  - If people can really see the why we'd get our shxx together and deal with stuff
  - Would make a big difference to have a common understanding / thread to our actions even if people focus on different issues (e.g. this person wants to engage with AI risk or this person with indigenous people and regeneration)
- Deep sensemaking (is necessary) for coherent action

### Hypothesis

- Deep sensemaking for people who want deep, rigorous understanding and who get that it isn't a simple explanation or a quick fix.
- "Explaining the [source of  the] metacrisis"

---

## Appendix: Different meta-crisis tribes of and the tower of babel of the second renaissance (paradigmatic changemakers)

A crude unfinished sketch ...

![image](https://github.com/life-itself/community/assets/180658/c8031db0-33c6-4f06-a06e-7a263f78cbc8)

**Comment by omri** [comment]
I would add SCQA to the curriculum as I've seen it used at the primary way to scope ideas and projects at Life Itself.

**Reply by rufuspollock** [reply]
@omri good point though this is more about the "tao" style way of doing things rather than contextual knowledge about how to make sense of the metacrisis - see https://scqa.lifeitself.org and https://datopian.com/playbook/scqh/

**Comment by laurenwigmore** [comment]
@rufuspollock you could share this with the individuals in the current residency for review and comment, it might be a good collective conversation point one evening. Along with maybe distilling this down into some opening questions that could be emailed out, that could lead into more reflections and/or follow up conversations/comments. 

Personally, I feel a 101 type course would be extremely useful and make a lot of the content more digestible for people and highlight what may be getting lost in the message in a more coherent way.
